tests/unit/test_proxysql_manager.py

```diff
@@ -53,3 +54,103 @@ def test__deregister_backend_does_not_delete_the_backend_if_not_registered():
 
     proxysql_man.deregister_backend(backend.hostgroup_id, backend.hostname,
                                     backend.port)
+
+
+def test_register_backend():
```

Comment:
Existing function names follow the convention of having the function name start with `test__` (two underscores), please use the same convention.